chirp.cc

```diff
@@ -0,0 +1,100 @@
+#include "sl_client.h"
```

Comment:
General comment: add unit tests for all functionality.

Reply:
added